File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/ForsteVirkningsdatoGrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;
import java.util.Date;

import no.nav.domain.pensjon.regler.PenPerson;
import no.nav.domain.pensjon.regler.kode.KravlinjeTypeCti;

public class ForsteVirkningsdatoGrunnlag implements Serializable {

    private Date virkningsdato;

    private Date kravFremsattDato;

    private PenPerson bruker;

    private PenPerson annenPerson;

    private KravlinjeTypeCti kravlinjeType;

    public ForsteVirkningsdatoGrunnlag() {
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/ForsteVirkningsdatoGrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.PenPerson
import no.nav.pensjon.regler.domain.enum.KravlinjeTypeEnum
import java.io.Serializable
import java.util.*
import java.time.LocalDate

class ForsteVirkningsdatoGrunnlag : Serializable {
    var virkningsdatoLd: LocalDate? = null
    var kravFremsattDatoLd: LocalDate? = null
    var bruker: PenPerson? = null
    var annenPerson: PenPerson? = null
    var kravlinjeTypeEnum: KravlinjeTypeEnum? = null
}